orion/pkg/repos/migrations/000007_create_table_sessions.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE sessions (
```

Comment:
If the users sql files are already `00007` you're going to have to rename your files `00008`. This is because if you change the ordering, everybody's migrations will be messed up. Always append!